cli/runtime/config/cli_discovery_sources_test.go

```diff
@@ -339,9 +342,46 @@ func TestDeleteCLIDiscoverySource(t *testing.T) {
 					},
 				},
 			},
+			count:   1,
 			input:   "test",
```

Comment:
test2?

Reply:
test case is valid description was misspelled. Corrected